### PR TITLE
Manage labels and CN23

### PR DESCRIPTION
**Pull request by damdam-s**
[IMP] refactor the response handler to be able to split the label and
eventually the CN23 returned by the Colissimo webservice

### CODE REVIEW

**Comment by hparfr** [comment]
Hi @damdam-s 

Thanks for your work.

I have few remarks : 
When output is set to pdf : the cn23 is in attachment and also included inside the pdf
When output is set to zpl :  the cn23 is only in attachment
In both cases cn23 is in pdf.

Because get_parts is tighly coupled with http headers, I think his place remains in Transport instead of Decode. What do you think ?

File: roulier/carriers/laposte/laposte_encoder.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def encode(self, api_input, action):
         sender_address = {
             "companyName": dj['from_address']['company'],
             "lastName": dj['from_address']['name'],
-            "firstName": "",
+            "firstName": dj['from_address'].get('firstname', ''),
```
Comment (hparfr) [inline roulier/carriers/laposte/laposte_encoder.py]: for consitency, please use dj['form_address']['firstname']

File: roulier/carriers/laposte/laposte_transport.py [hunk 2]
```diff
@@ -108,7 +108,6 @@ def extract_body(response):
         status = self.STATUS_ERROR
         payload = None
 
-        print "voila le pyalod", extract_payload(response_xml)
```
Comment (hparfr) [inline roulier/carriers/laposte/laposte_transport.py]: 👍

**Comment by damdam-s** [comment]
@hparfr fix done

**Comment by hparfr** [comment]
:+1: 
Thanks !

**Pull request merged**